flashinfer/comm.py

```diff
@@ -238,6 +238,11 @@ def gen_comm_module() -> JitSpec:
             jit_env.FLASHINFER_CSRC_DIR / "flashinfer_comm_ops.cu",
             jit_env.FLASHINFER_CSRC_DIR / "custom_all_reduce.cu",
             jit_env.FLASHINFER_CSRC_DIR / "trtllm_allreduce.cu",
+            jit_env.FLASHINFER_CSRC_DIR / "trtllm_alltoall.cu",
```

Comment:
It's better to decouple the modules into more atomic ones, e.g. `gen_alltoall_comm_module` etc.